docker-compose.override.yaml

```diff
@@ -13,3 +13,12 @@ services:
     volumes:
       - ./app:/home/node/app
       - ./package.json:/home/node/package.json
+
+  ffc-mpdp-backend-postgres:
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+volumes:
+  postgres_data:
```

Comment:
Exposing a port for local dev so client tools like pgAdmin, Azure Data Studio etc can connect.

Also volume binding data so it can be persisted locally between container restarts.